nx/include/switch/services/ncm_types.h

```diff
@@ -148,3 +170,22 @@ typedef struct {
     u8 storageID;           ///< \ref NcmStorageId
     u8 pad[7];              ///< Padding
 } NcmProgramLocation;
+
+/**
+ * @brief Converts the size fields from an input \ref NcmContentInfo into a usable 64-bit integer.
+ * @param[in] info Pointer to \ref NcmContentInfo object.
+ * @param[out] out Output size.
+ */
+NX_INLINE void ncmContentInfoSizeToU64(const NcmContentInfo *info, u64 *out) {
```

Comment:
`NX_CONSTEXPR u64 ncmContentInfoSizeToU64(const NcmContentInfo *info)`

Reply:
Care to explain what you mean by "more straightforward"?

Reply:
Something along the lines of `Retrieves the content size from an input \ref NcmContentInfo stuct`.

Reply:
Got it, thanks.